.circleci/config.yml

```diff
@@ -28,15 +28,15 @@ jobs:
      
       - run:
           name: Collapse RSEM (polyA RNA-seq)
-          command: ./scripts/run_in_ci.sh Rscript analyses/collapse-rnaseq/01-summarize_matrices.R -i data/pbta-gene-expression-rsem-fpkm.polya.rds -g analyses/collapse-rnaseq/gencode.v27.primary_assembly.annotation.gtf.gz -m data/pbta-gene-expression-rsem-fpkm-collapsed.polya.rds -t analyses/collapse-rnaseq/pbta-gene-expression-rsem-fpkm-collapsed-table.polya.rds
+          command: ./scripts/run_in_ci.sh Rscript analyses/collapse-rnaseq/01-summarize_matrices.R -i data/pbta-gene-expression-rsem-fpkm.polya.rds -g data/gencode.v27.primary_assembly.annotation.gtf.gz -m data/pbta-gene-expression-rsem-fpkm-collapsed.polya.rds -t analyses/collapse-rnaseq/pbta-gene-expression-rsem-fpkm-collapsed_table.polya.rds
```

Comment:
> Thank you for adding this analysis!
> 
> I was surprised at first that there were such low correlations, but I would guess that those are for genes that are very lowly expressed, and therefore most of the variation we are seeing in them is noise. I looked briefly to see if there was much difference if we switched to Spearman correlation, and it might improve some correlations a bit, but not a drastic change.
> 
> My one major comment is about the location of the output files. While we will be including these files in the data release whcih will put them in the `data/` folder, we don't really want analysis scripts to write there directly, so as to preserve that a "protected" location for standardized files. If you write the output files to the analysis folder (or a subfolder, as [discussed here](https://github.com/AlexsLemonade/OpenPBTA-analysis#folder-structure)), you don't necessarily have to include all of them in the tracked repo (either by just not adding them or making a `.gitignore` file in the analysis folder).
> 
> On a related note: the `analyses/collapse-rnaseq/` folder still contains what appear to be the old versions of `pbta-gene-expression-rsem-fpkm-collapsed-table.*` files? If so, it would be good to update those with the latest versions in this PR. You can also delete the gtf file from this folder, as are now using the version in the `data/` folder.
> 
> Let me know if that all makes sense to you. Thanks again for your contributions!

@jashapiro Thank you for getting back so quickly, and makes sense regarding not writing out to `data/`, will make those changes to config.yml and 01-summarize_matrices.R.